dataedit/helper.py

```diff
@@ -54,6 +55,15 @@ def get_readable_table_name(table_obj: Table) -> str:
     return label
 
 
+##############################################
+#            Metadata related                #
+##############################################
+
+def order_metadata(metadata: dict) -> dict:
```

Comment:
👍 If you agree that this is good enout. There is still a slight difference because this only sorts the "top" level buttons. I think it's fine.

Reply:
could adapt this also, so that key order is preserved recursively

Reply:
Right, that could be a quick solution. Then I would vote for implementing it : in favour of everything being as similar as possible.